contrib/postgresql/template.py

```diff
@@ -0,0 +1,106 @@
+pkgname = "postgresql"
+pkgver = "16.2"
+pkgrel = 0
+build_style = "configure"
```

Comment:
meson

even if it wasn't, this is a gnu_configure project too. but meson is better here